src/Ng/Jobs/BackupV3Storage.cs

```diff
@@ -0,0 +1,236 @@
+
+                    Logger.LogInformation($"StartContainerCopy:{container}");
+                    sw.Start();
+                    Process p = Process.Start(copyToAzureProc);
+                    p.WaitForExit();
+#if Debug
+                string result = p.StandardOutput.ReadToEnd();
+                Console.Write(result);
+#endif
+                    sw.Stop();
+                    int exitCode = p.ExitCode;
+                    Logger.LogInformation($"EndContainerCopy:{container}:{exitCode}:{sw.ElapsedMilliseconds}");
+                    p.Close();
+                }
+                catch (Exception ex)
+                {
+                    Logger.LogError($"BackupV3Storage:Exception on backup save.{ex.ToString()}.");
+                    return false;
+                }
+                try
+                {
+                    ProcessStartInfo copyToAzureProcLog = new ProcessStartInfo();
+                    copyToAzureProcLog.FileName = @"cmd.exe";
+                    copyToAzureProcLog.Arguments = $"/c {AzCopyPath} {argumentsLog}";
+                    copyToAzureProcLog.CreateNoWindow = false;
+                    copyToAzureProcLog.UseShellExecute = false;
+                    Process pLog = Process.Start(copyToAzureProcLog);
+                    pLog.WaitForExit();
+                    pLog.Close();
+                }
+                catch (Exception ex)
+                {
+                    Logger.LogError($"BackupV3Storage:Exception on log save.{ex.ToString()}.");
+                }
+
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        private void RefreshLogData(string logFolder, string logFile)
+        {
+            if(Directory.Exists(logFolder))
+            {
+                Directory.Delete(logFolder, true);
+            }
+            Directory.CreateDirectory(logFolder);
+            var stream = File.Create(logFile);
+            stream.Close();
+        }
+
+        private async Task<bool> TryCreateDestinationContainerAsync(string containerName, string storageAccountName, string storageAccountKey)
+        {
+            CloudBlobClient blobClient = GetCloudBlobClient(storageAccountName, storageAccountKey);
+
+            try
+            {
+                CloudBlobContainer container = blobClient.GetContainerReference(containerName);
+                await container.CreateIfNotExistsAsync();
+                return true;
+            }
+            catch(Exception ex)
+            {
+                Logger.LogError($"BackupV3Storage:Exception on create container {containerName}.{ex.ToString()}.");
+                return false;
+            }
+        }
+
+        private async Task<bool> TryDeleteContainerAsync(string containerName, DateTime date, string storageAccountName, string storageAccountKey)
+        {
+            string destContainer = $"{Format(date.AddDays(-1* _backupDays))}-{containerName}";
```

Comment:
Added support to retain only fix number of containers . 
PR: https://github.com/NuGet/NuGet.Jobs/pull/151